cmd/crictl/attach.go

```diff
@@ -43,6 +43,17 @@ var runtimeAttachCommand = &cli.Command{
 			Aliases: []string{"i"},
 			Usage:   "Keep STDIN open",
 		},
+		&cli.StringFlag{
+			Name:    "transport-protocol",
+			Aliases: []string{"p"},
+			Usage:   "protocol of transport, One of: spdy|websocket",
+			Value:   "spdy",
+		},
+		&cli.BoolFlag{
```

Comment:
why a flag and not using fall back , as we are doing with kubectl and is defined in the KEP?

flags are hard to remove in the future and become APIs for users, I also prefer an environment variable instead

Reply:
thk you reply. fixed